modules/rest/src/main/java/com/enonic/xp/app/contentstudio/rest/resource/content/OutboundDependenciesIds.java

```diff
@@ -0,0 +1,27 @@
+package com.enonic.xp.app.contentstudio.rest.resource.content;
+
+import com.enonic.xp.content.ContentIds;
+
+class OutboundDependenciesIds
+{
+    private final ContentIds existingOutboundIds;
+
+    private final ContentIds nonExistingOutboundIds;
+
+    public OutboundDependenciesIds( final ContentIds existingIds, final ContentIds nonExistingIds )
```

Comment:
```suggestion
   OutboundDependenciesIds( final ContentIds existingIds, final ContentIds nonExistingIds )
```
no need in excessive `public` access level in non-public class